### PR TITLE
Adds Kotlin multiplatform case study

### PR DESCRIPTION
**Pull request by KarenBonillaM**
Hi!
 Here's my case study about KMP. Please let me know if you have any feedback.

### CODE REVIEW

File: src/content/docs/kotlin-multiplatform/kotlin-multiplatform-case-study.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+**Expendability**
+Kotlin Multiplatform allows developers to create a common API for platform-specific functionalities, offering more flexibility than flutter.
+
+**Language**
+Kotlin is more popular and loved according to the Stack Overflow Survey 2022, providing better compile-time safety.
+
+**Supported Platforms**
+KMP was initially designed for Android and iOS, with evolving support for browsers and backends. Flutter targets a broader range of platforms using the same codebase.
+
+## Getting Started
```
Comment (sulenchy) [inline src/content/docs/kotlin-multiplatform/kotlin-multiplatform-case-study.md:76]: @KarenBonillaM Well done on this write-up. However, this section is hard to follow up. Imagine someone new to programming reading this. I hereby  recommend you show some illustration.

**Review by KarenBonillaM — COMMENTED** [review]
Hi Abi!

I have made some changes.
Please let me know if I need to add something else

File: src/content/docs/kotlin-multiplatform/kotlin-multiplatform-case-study.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+**Install Kotlin**
+
+Download and install one of these IDEs to start using Kotlin.
+
+- https://www.jetbrains.com/idea/download/?_gl=1*kv902d*_ga*MTM0NjMyMjQwNy4xNzA3NzQ1MzI2*_ga_9J976DJZ68*MTcwODE2ODM5My4yLjEuMTcwODE2ODU0MC40Ny4wLjA.&_ga=2.21385277.701422824.1708168394-1346322407.1707745326&section=mac
+- https://developer.android.com/studio
```
[the remark this answers is not in the document]
Comment (sulenchy) [reply src/content/docs/kotlin-multiplatform/kotlin-multiplatform-case-study.md:83]: LGTM

**Review by sulenchy — APPROVED** [review]
Looks good to me now
